WcaOnRails/app/controllers/polls_controller.rb

```diff
@@ -3,7 +3,7 @@
 class PollsController < ApplicationController
   before_action :authenticate_user!
   before_action -> { redirect_to_root_unless_user(:can_create_poll?) }, only: [:new, :create, :update, :destroy]
-  before_action -> { redirect_to_root_unless_user(:can_vote_in_poll?) }, only: [:index, :vote, :results]
+  before_action -> { redirect_to_root_unless_user(:can_view_poll?) }, only: [:index, :results]
```

Comment:
This looks like it's from your other PR, can you remove it from this PR?

Reply:
Done.